test/Microsoft.Azure.Devices.Edge.Test.Common/linux/CertHelper.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace Microsoft.Azure.Devices.Edge.Test.Common.Linux
+{
+    using System.IO;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices.Edge.Test.Common.Certs;
+    using Serilog;
+
+    public static class CertHelper
+    {
+        public static async Task<EdgeCertificates> GenerateEdgeCertificatesAsync(string deviceId, string scriptPath, CancellationToken token)
+        {
+            var command = $"'{scriptPath}' create_edge_device_certificate '{deviceId}'";
+
+            await Profiler.Run(
+                async () =>
+                {
+                    string[] output =
+                        await Process.RunAsync("bash", $"-c \"{command}\"", token);
+                    Log.Verbose(string.Join("\n", output));
+                },
+                "Created certificates for edge device");
+
+            string dir = new FileInfo(scriptPath).DirectoryName;
+
+            return new EdgeCertificates(
```

Comment:
should we check these files exists here to ensure the script file generate them as expected?

Reply:
Yeah probably.

Reply:
Done. Consolidated the implementations of the platform-specific functions that generate cert files at well-known locations, and added the checks in there (see the `NormalizeFiles` function).